back_end/RayTracer/Models/Json/EnvironmentJSON.cs

```diff
@@ -15,15 +15,15 @@ public class EnvironmentJSON
         public WindowFrame winFrame;
         public ColorRGB background;
         public List<Light> lights;
-        public Camera camera;
+        public Perspective camera;
```

Comment:
In this change, our program will just have perspective camera, so we made it specific to be perspective instead of camera.